src/esbmc/options.cpp

```diff
@@ -435,7 +435,9 @@ const struct group_opt_templ all_cmd_options[] = {
     // Abort if the program contains a recursion
     {"abort-on-recursion", NULL, ""},
     // Verbosity of message, probably does nothing
-    {"verbosity", boost::program_options::value<int>(), ""},
+    {"verbosity",
+     boost::program_options::value<std::vector<std::string>>(),
+     ""},
```

Comment:
we can probably add more help info in line 440, something like "module:N to get the N-level debug messages from a specific module"

Reply:
Done.